app/src/main/kotlin/co/netguru/baby/monitor/client/application/di/ApplicationModule.kt

```diff
@@ -84,8 +84,8 @@ object ApplicationModule {
 
     private val MIGRATION_4_5 = object : Migration(4, 5) {
         override fun migrate(database: SupportSQLiteDatabase) {
-            database.execSQL("ALTER TABLE CHILD_DATA ADD COLUMN noiseSensitivity INTEGER NOT NULL DEFAULT ${NoiseDetector.DEFAULT_NOISE_SENSITIVITY}")
-            database.execSQL("ALTER TABLE CLIENT_DATA ADD COLUMN noiseSensitivity INTEGER NOT NULL DEFAULT ${NoiseDetector.DEFAULT_NOISE_SENSITIVITY}")
+            database.execSQL("ALTER TABLE CHILD_DATA ADD COLUMN noiseLevel INTEGER NOT NULL DEFAULT ${NoiseDetector.DEFAULT_NOISE_LEVEL}")
+            database.execSQL("ALTER TABLE CLIENT_DATA ADD COLUMN noiseLevel INTEGER NOT NULL DEFAULT ${NoiseDetector.DEFAULT_NOISE_LEVEL}")
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/bf6320090a88ce6cc95c72470de2d4697673ea27/app/src/main/kotlin/co/netguru/baby/monitor/client/application/di/ApplicationModule.kt#L88"></span>Line detected that is longer than the defined maximum line length in the code style.</td>
    </tr>
  </tbody>
</table>